mobile-app/cypress/integration/functional/wallet/balances/balances.spec.ts

```diff
@@ -28,22 +40,50 @@ context('Wallet - Balances', () => {
     cy.getByTestID('dfi_token_label').contains('Token')
     cy.getByTestID('total_dfi_amount').contains('20.00000000')
     cy.getByTestID('total_dfi_label').contains('DFI')
+    cy.wrap(whale.prices.getFeedActive('DFI', 'USD')).then((dfiResponse) => {
```

Comment:
i think you can refactor this into a recursive function, skip `cy.checkBalanceRow()` because they will be asserted in the next test `it('should display BTC and ETH with correct amounts'`

Main purpose here is to flatten the logic, avoid nesting multiple layers, easier to maintain and read